src/backend/function/Fusion.Resources.Functions/Functions/ProfileSync.cs

```diff
@@ -22,6 +24,24 @@ public ProfileSync(IHttpClientFactory httpClientFactory)
             resourcesClient = httpClientFactory.CreateClient(HttpClientNames.Application.Resources);
         }
 
+        [FunctionName("profile-sync-event")]
+        public async Task SyncProfile(
+            [EventSubscriptionTrigger(HttpClientNames.Application.People, "subscriptions/persons", "resources-profile")] MessageContext message,
+            ILogger log)
+        {
+            log.LogInformation("Profile sync event received");
+            log.LogInformation(message.Event.Data);
+            var body = message.GetBody<PeopleSubscriptionEvent>();
+
+            if (body.Type != PeopleSubscriptionEventType.ProfileUpdated) //manual filtering to avoid hard-coding in attribute
```

Comment:
The property checked here is not available for filtering when setting up the event subscription. So the comment can be removed as misleading?

Reply:
Ok, my misunderstanding, will remove.
I am quite confused as to what is the difference between "PeopleEventTypes" and "PeopleSubscriptionEventType". Perhaps other users might be as well, and some docs/comments on this is in place?

Reply:
The difference is that one type is on the subscription, the other is to fine grain the event.. To use filters on the service bus subscription, the property needs to be promoted to the service bus message headers. To do that it needs to be generalized in some sort --> SubscriptionEventType..  

But inside the message there might be more details, to not have too many changes in the subscription event constants, but one might still want to determine if the event should be acted upon etc.. 

Subscription type: `Profile`, `Companies`
PeopleEventType: `ProfileUpdated`, `ProfileDeleted`, `GPRDClearoutsale`, `AccountSignin`
CompanyEventType: `CompanyUpdated`, `CompanyCreated`, `CompanyDeprecated` etc... 

If the consuming service have no use for companies, the subscription can be filtered so no cpu is spent on useless messages.